.clang-format

```
@@ -67,7 +67,7 @@ PenaltyBreakString: 70
 PenaltyBreakTemplateDeclaration: 0
 PenaltyExcessCharacter: 100
 PenaltyReturnTypeOnItsOwnLine: 90
-PointerAlignment: Right
+PointerAlignment: Left
```

This should be a different discussion. One bikeshed at a time please :)

```suggestion
PointerAlignment: Right
```

---

> Interesting -- this has been an annoyance for so long!
> 
> I'm curious why we'd need the macro. If `__host__` and `__device__` are both marked as statement macros, `__host__ __device__` should work too, right?

No, because we tell `clang-format` that `__host__` is a statement so it will change

---

Ouch, yeah, that's rough. Maybe `AttributeMacros` would behave better? I'd like to avoid adding a special macro here if possible.